web/src/tools/testing/bulk_rendering/renderer_core.ts

```diff
@@ -108,7 +112,16 @@ export class BatchRenderer {
 
       eleDescription.appendChild(document.createTextNode('Name: ' + kbd.Name));
       eleDescription.appendChild(document.createElement('br'));
-      eleDescription.appendChild(document.createTextNode('Font:  ' + keyman.core.activeKeyboard['_legacyLayoutSpec'].F));
+
+      const keyboard = keyman.core.activeKeyboard;
+      // Some keyboards, such as sil_euro_latin and sil_ipa, no longer specify this property.
```

Comment:
sil_euro_latin has never specified the `KV` property (which is what `_legacyLayoutSpec` references). It's always been optional. So I am puzzled why this was working in the past.

Here's what the earliest version of sil_euro_latin (1.8) provides for `KV`:

```
this.KV=null;
```